### DISCUSSION TITLE
OpenSCAP security guide

### DISCUSSION THREAD

**Discussion by SLSAKrit** [ComplianceAsCode/content#12679] [Q&A] [answered]
Hi,

I'm new to OpenSCAP. I've been given the task of evaluating compliance of our application. I was exploring MITRE SAF automation and OpenSCAP. In OpenSCAP, I'm able to get security guides for operating systems. But, I'm unable to see any security guides for webservers or databases. Does OpenScap at present have guidance only for the operating systems or am I the one unable to figure out the location of the guides for other software components like webservers or databases? Please help me.

Thanks,
Krithika

**Comment by dodys** [comment, marked as answer]
Currently the project only contains data about Operating Systems, not for other benchmarks targets.
